configuration-example.yaml

```diff
@@ -66,7 +66,7 @@ calendarAPI:
   exceptionTimezone: America/Boise #for certain campuses or locations, a different timezone is used
   exceptionTimezoneCampusID: 110 #campus ID that uses the exception timezone
 
-httpClient:
+httpClientConfiguration:
```

Comment:
I needed this change in my configuration for a successful build - @ianbrown9475  confirmed that he had to as well.